packages/aspell.rb

```diff
@@ -7,23 +7,10 @@ class Aspell < Package
   source_url 'ftp://alpha.gnu.org/gnu/aspell/aspell-0.60.7-rc1.tar.gz'
```

Comment:
Bump version?

Reply:
It is bumped ([link](https://github.com/yancouto/chromebrew/blob/aspell_fix/packages/aspell.rb#L6))

Reply:
Maybe it needs bumped again?  It's not showing up as a change on the diff.

Reply:
Ohhh, I had bumped it before merging the binaries commit, and it was bumped there too, haha. Will change it.